src/RestfulServer.php

```diff
@@ -59,17 +80,18 @@ class RestfulServer extends Controller
      */
     protected $member;
 
-    public static $allowed_actions = array(
-        'index'
+    private static $allowed_actions = array(
+        'index',
+        'notFound'
     );
 
     /*
     function handleItem($request) {
-        return new RestfulServer_Item(DataObject::get_by_id($request->param("ClassName"), $request->param("ID")));
+        return new RestfulServerItem(DataObject::get_by_id($request->param("ClassName"), $request->param("ID")));
     }
 
     function handleList($request) {
-        return new RestfulServer_List(DataObject::get($request->param("ClassName"),""));
+        return new RestfulServerList(DataObject::get($request->param("ClassName"),""));
     }
     */
```

Comment:
Later: We may as well nuke commented out code

Reply:
I was going to, but decided against these two as they relate to a @todo and would greatly improve the overall structure of the code (less sniffing if we have/want lists or single objects, and conditionals all through to deal with each).

I guess it was more of a reminder to open an issue, which we can then address later. I'll do that now, then remove the comments.